cmd/kube-plex/metadata.go

```diff
@@ -133,6 +136,16 @@ func (p PmsMetadata) ResourceRequirements() corev1.ResourceRequirements {
 	}
 }
 
+// getNodeSelector from pod spec based on the selectors given
+func getNodeSelector(nodeSelector, defname string, pod *corev1.Pod) (map[string]string) {
+	n := pod.Spec.NodeSelector
+	if n != nil {
+                return n
+	} else {
+		return  map[string]string{"kubernetes.io/arch": "amd64"}
```

Comment:
We could actually just return an empty map here. The implicit default might be a bit surprising to folks. I think this was lifted directly from the original kube-plex codebase and I just never thought this through.

Reply:
Yeah, I can drop that when I get back from lunch.

Reply:
Okay, changed it to an empty map object.